templates/index.pug

```diff
@@ -1,3 +1,112 @@
 extend layouts/layout.pug
 block Main
-  .home day la home
+  section.section-title
+    .overlay
+    .section-container
+      h1.text-bold Vegan Store Simple Website Template
+      p With nutrition, all the vital functions of the body are connected. It is the source of the development of tissues and cells, their constant renewal, the saturation of man with energy.
+
+  section.feature
+    .feature-card
+      .card.card-1
+        .card-wrapper
+          .card-image
+            span(class="mbr-iconfont fa fa-cut")
+          .cartd-title Vegetables are an integral part of the human diet. They are extremely useful because of the high content of carbohydrates, various acids, vitamins and active elements in the form, easy for digestion.
```

Comment:
```suggestion
          .feature__title Vegetables are an integral part of the human diet. They are extremely useful because of the high content of carbohydrates, various acids, vitamins and active elements in the form, easy for digestion.
```